docs/source/model/dependencies.rst

```diff
@@ -0,0 +1,453 @@
+Managing Dependencies in MLflow Model
+=====================================
+
+`MLflow Model <../models.html>`_ is a standard format that packages a machine learning model with its dependencies and other metadata.
+Building a model with its dependencies allows for reproducibility and portability across a variety of platforms and tools.
+
+When you create an MLflow model using the `MLflow Tracking APIs <../tracking.html>`_, for instance, :py:func:`mlflow.pytorch.log_model`,
+MLflow automatically infers the required dependencies to your model flavor and records them as a part of Model metadata. Then when you
+serve the model for prediction, MLflow automatically installs the dependencies to the environment. Thus, normally you don't need to
+worry about managing dependencies in MLflow Model.
+
+However, in some cases, you may need to add or modify some dependencies. This page provides a high-level description of how MLflow manages
+dependencies and guidance for how to customize dependencies for your use case.
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 1
+
+.. _how-mlflow-records-dependencies:
+
+How MLflow Model Records Dependencies
+-------------------------------------
+
+MLflow Model is saved in a directory with the following structure:
+
+::
+
+    my_model/
+    ├── MLmodel
+    ├── model.pkl
+    ├── conda.yaml
+    ├── python_env.yaml
+    └── requirements.txt
+
+Model dependencies are defined by the following files (For other files, please refer to :ref:`Storage Format <model-storage-format>`):
+
+* ``python_env.yaml`` - This file contains the information required to restore the model environment using virtualenv (1) python version (2) build tools like pip, setuptools, and wheel (3) pip requirements of the model (a reference to requirements.txt)
```

Comment:
Are the numbers here to indicate positions in a diagram you'll be adding?

Reply:
nope just for listing (was a bullet points but just changed to make it use lesser space)

Reply:
That's a cool feature that I didn't know existed! Neat!